core-bundle/src/EventListener/Menu/BackendFavoritesListener.php

```diff
@@ -151,7 +151,7 @@ private function buildHeaderMenu(MenuEvent $event, BackendUser $user): void
         $event->getTree()->addChild($tree);
 
         // Move the favorites menu behind "manual"
-        (new MenuManipulator())->moveToPosition($tree, 1);
+        (new MenuManipulator())->moveToPosition($tree, 0);
```

Comment:
```suggestion
        (new MenuManipulator())->moveToFirstPosition($tree);
```

There is a dedicated method to move the item to the first position 😄

Reply:
See 3925c0af795f61c82d94fc8a9e9e7a993cff4dbd